presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -600,6 +601,18 @@ public static long random(@SqlType(StandardTypes.BIGINT) long value)
         return ThreadLocalRandom.current().nextLong(value);
     }
 
+    @Description("a pseudo-random number from a normal distribution with mean, std, and p")
+    @ScalarFunction(alias = "rand_normal", deterministic = false)
```

Comment:
A Java function named `randomNormal` will be converted to the snake-case SQL function `random_normal`, unless `value = ...` is specified for the annotation. `alias` is used to add additional aliases.

So this definition creates both `random_normal()` and `rand_normal()`. Is that intentional? I'm not sure we need both versions -- probably better to just pick one. Will leave that to @martint to decide